lib/config.js

```diff
@@ -104,7 +105,23 @@ function normalizeModulesConfig(modules) {
 	return modulesConfig;
 }
 
+function normalizeMinifierConfig(config) {
+	// If minifyCSS is false or a function, we don't need to touch it
+	if (config && (!config.minifyCSS || typeof config.minifyCSS === 'function')) {
```

Comment:
Lets alter this check:
```js
if (!config || !config.minifyCSS || typeof config.minifyCSS === 'function') {
  return config;
}
```

This way the rest of the function can assume config is defined.  I checked `html-minifier` uses `options || {}` so passing undefined is not harmful (although I'm not sure what the point would be).

Reply:
Additional details on this, your current patch causes a significant change to the behavior when the `htmlMinifier` option is not provided to the babel plugin.  In the current release this results in html-minifier not calling `clean-css` at all.  With these current changes not providing `htmlMinifier` results in calling `clean-css` with an undefined options object (specific example the default optimization level is 2).

Reply:
That's a good point. We could put this functionality behind a flag, `minifyPartialCSS` or something like that?

Reply:
Do you have a chance to discuss this one on slack?

Reply:
My starting position for this is that if people want CSS to be minified then they should provide a minimum configuration `{modules: {...}, htmlMinifier:{minifyCSS: {}}}`.  Probably need to alter the `.baberc` displayed in the README to include `minifyCSS: {}`, it currently implies that this is not needed for CSS minification which has never been true.

Reply:
I made this so that `htmlMinifier: { minifyCSS: true }` works as well, using the default configuration